packages/mikro-orm/src/index.ts

```diff
@@ -0,0 +1,148 @@
+import { RequestContext, wrap } from "@mikro-orm/core"
+
+import {
+  Account,
+  Session,
+  User as DefaultUserModel,
+  VerificationToken,
+} from "./models"
+import {
+  Adapter,
+  AdapterSession,
+  VerificationToken as AdapterVerificationToken,
+} from "next-auth/adapters"
+import { DefaultAccount } from "next-auth"
```

Comment:
```suggestion
import type {
  Adapter,
  AdapterSession,
  VerificationToken as AdapterVerificationToken,
} from "next-auth/adapters"
import type { DefaultAccount } from "next-auth"
```

Let's mark type imports with the `type` keyword for clarity.

This is true for all the places, but I will only comment here.

Reply:
Hi! I'd introduce a linting rule for that. But I already saw that not all adapter comply to it, so I wont make it an Error for now.